bigbluebutton-html5/public/locales/en.json

```diff
@@ -623,7 +623,9 @@
     "app.toast.meetingMuteOn.label": "All users have been muted",
     "app.toast.meetingMuteOff.label": "Meeting mute turned off",
     "app.toast.setEmoji.raiseHand": "You have raised your hand",
-    "app.toast.setEmoji.lowerHand": "Your hand has been lowered",
+    "app.toast.setEmoji.lowerHand": "You have lowered your hand",
```

Comment:
Please consider reverting this change. This wording has been changed intentionally, see #13301 for reasons.